From 195cf40eb3d8fc606c89d619b508cf63749a121d Mon Sep 17 00:00:00 2001
From: Ondrej Holy <oholy@redhat.com>
Date: Fri, 7 Jul 2023 08:52:04 +0200
Subject: [PATCH] fuse: Include locale.h header

The commit eeb9914c breaks the build in some cases. This is because
the `setlocale` function is called, but the locale.h header file is
not included. Let's include it to fix this issue.

Fixes: https://gitlab.gnome.org/GNOME/gvfs/-/issues/681


(cherry picked from commit 84a98b56ce7e46f1e10238e09e5c6470c54dbaa9)
---
 client/gvfsfusedaemon.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/client/gvfsfusedaemon.c b/client/gvfsfusedaemon.c
index 79081cdf..1fcaf614 100644
--- a/client/gvfsfusedaemon.c
+++ b/client/gvfsfusedaemon.c
@@ -32,6 +32,7 @@
 #include <string.h>
 #include <errno.h>
 #include <signal.h>
+#include <locale.h>
 
 #include <glib.h>
 #include <glib/gi18n.h>
-- 
GitLab

